modular_skyrat/modules/modular_items/code/bags.dm

```diff
@@ -1,40 +1,130 @@
 	atom_storage.max_total_storage = 30
 	atom_storage.max_slots = 3
 	atom_storage.numerical_stacking = FALSE
 	atom_storage.can_hold = typecacheof(list(/obj/item/ammo_box/magazine, /obj/item/ammo_casing, /obj/item/ammo_box/revolver, /obj/item/stock_parts/cell/microfusion))
 
-/obj/item/storage/bag/material
+/obj/item/storage/pouch/material
 	name = "material pouch"
 	desc = "A pouch for sheets and RCD ammunition that manages to hang where you would normally put things in your pocket."
 	icon = 'modular_skyrat/modules/modular_items/icons/storage.dmi'
 	icon_state = "materialpouch"
-	slot_flags = ITEM_SLOT_POCKETS
 	w_class = WEIGHT_CLASS_BULKY
-	resistance_flags = FLAMMABLE
 	custom_price = PAYCHECK_CREW * 4
 
-/obj/item/storage/bag/material/Initialize(mapload)
+/obj/item/storage/pouch/material/Initialize(mapload)
 	. = ..()
 	atom_storage.max_specific_storage = WEIGHT_CLASS_NORMAL
 	atom_storage.max_total_storage = INFINITY
 	atom_storage.max_slots = 2
 	atom_storage.numerical_stacking = TRUE
 	atom_storage.can_hold = typecacheof(list(/obj/item/rcd_ammo, /obj/item/stack/sheet))
 
-/obj/item/storage/bag/trash
-	slot_flags = ITEM_SLOT_BELT //QoL by Gandalf
-	worn_icon_state = "trashbag"
-	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/belt.dmi'
+/// It's a pocket medkit. Use sparingly?
+/obj/item/storage/pouch/medical
+	name = "medkit pouch"
+	desc = "A standard medkit pouch compartmentalized for the bare essentials of field medical care, made with fireproof kevlar \
+	(but hopefully you never have to test that). Cannot, itself, contain a medkit. Comes with a pocket clip."
+	resistance_flags = FIRE_PROOF
+	icon_state = "medkit"
+	/// The list of things that medical pouches can hold. Stolen from what medkits can hold, but modified for things you would probably want at pocket-access.
+	var/static/list/med_pouch_holdables = list(
+		/obj/item/healthanalyzer,
+		/obj/item/dnainjector,
+		/obj/item/reagent_containers/dropper,
+		/obj/item/reagent_containers/cup/beaker,
+		/obj/item/reagent_containers/cup/bottle,
+		/obj/item/reagent_containers/cup/tube,
+		/obj/item/reagent_containers/pill,
+		/obj/item/reagent_containers/syringe,
+		/obj/item/reagent_containers/medigel,
+		/obj/item/reagent_containers/spray,
+		/obj/item/reagent_containers/hypospray,
+		/obj/item/storage/pill_bottle,
+		/obj/item/stack/medical,
+		/obj/item/flashlight/pen,
+		/obj/item/bonesetter,
+		/obj/item/cautery,
+		/obj/item/hemostat,
+		/obj/item/reagent_containers/blood,
+		/obj/item/stack/sticky_tape,
+	)
+
+/obj/item/storage/pouch/medical/Initialize(mapload)
+	. = ..()
+	atom_storage.max_specific_storage = WEIGHT_CLASS_NORMAL
+	atom_storage.max_slots = 7
+	atom_storage.max_total_storage = 14
+	atom_storage.set_holdable(med_pouch_holdables)
+
+/obj/item/storage/pouch/medical/loaded/Initialize(mapload)
+	. = ..()
+	var/static/items_inside = list(
+		/obj/item/stack/medical/gauze/twelve = 1,
+		/obj/item/stack/medical/splint = 1,
```

Comment:
Splints aren't a thing anymore (even if I don't think that was necessarily the good thing to do), so you'll have to remove them from here.

Reply:
oh yeah i forgot that pr got merged